src/cdomains/valueDomain.ml

```diff
@@ -612,6 +624,15 @@ struct
         end
       | _ -> BinOp(MinusPP, exp, StartOf start_of_array_lval, intType)
     in
+    (* Create a typesig from a type, but drop the arraylen attribute *)
+    let typeSigWithoutArraylen t =
```

Comment:
`typeSigWithoutArrayLen`?

Reply:
Since the attribute we are removing is actually `Attr("arraylen",_)`, I'd stick with non camelCase version here personally, but if you feel strongly about having it camelCase, I'm fine with that too